src/Application/Persistence/RetrySafeEntityManager.php

```diff
@@ -167,14 +167,14 @@ public function refresh($object, ?int $lockMode = null): void
     public function flush($entity = null): void
     {
         try {
-            $this->entityManager->flush($entity);
+            $this->entityManager->flush();
```

Comment:
Seems counterintuitive to keep the method sig aligned with `EntityManagerDecorator`, including the `$entity`, but then do nothing with it. If there's some advantage might be good to explain the method doc block?

Reply:
This was already effectively doing nothing with the entity I think, since `\Doctrine\Persistence\ObjectManager::flush` doesn't take any parameters. Psalm reported it as an error. 

But I see thanks to your comment that the decorator interface does take the param and so does the concrete EntityManager from Doctrine. If passed that triggers the deprecation message with a link to https://github.com/doctrine/orm/issues/8459 . So I guess passing the entity doesn't do nothing as I thought, its just not recommended. 

We never pass anything when calling `flush`. Not sure why Doctrine removed the param from the interface before removing it from the decorator and the concrete class. I think I'll put it back here for now.

Reply:
Done in d53b31dbec4acf01fd